### ISSUE TITLE
KYTPersistenceTests.m

### ISSUE THREAD

**Issue by MarkShenT** [LukePerficient/KYT#4]
- (void)testReadDataFromFile {
    NSString *newFilePath = @"TeamMember.plist";
    NSString *docfilePath = [KYTTeamMemberPersistence getDocPath:newFilePath];
    
    NSMutableArray *linearArray = [[NSMutableArray alloc] initWithContentsOfFile:docfilePath];
    
    for (int i = 0; i < [linearArray count]; i++) {
        NSLog(@"%@",linearArray[i]);
    }
}

meaningless test method

**Comment by Duskamo** [comment]
Resolved, will push to repository tomorrow on 9/19. Will post another comment with commit id.

**Comment by Duskamo** [comment]
Can view the latest commit f17e0d7cbacadd8d589b77d2c0b380a4bddab9d7 for checking resolved issue.

**Comment by MarkShenT** [comment]
@Duskamo  I didn't find any related changes in  f17e0d7, please double check if that's the correct commit number

**Comment by Duskamo** [comment]
Wow github is cool being able to reference users and commits through comments.

@MarkShenT I fixed three issues and pushed them all at once under the commit number f62da3d42e61f01c4f82c1a8465b82589e821e2b referenced above. I used f17e0d7cbacadd8d589b77d2c0b380a4bddab9d7 as my last comment because it was the last push of the day.

However from now on when I push my issue fixes to github I'll just keep it one-to-one like we talked about the other day, with one issue fixed in one commit to keep it easy to track for you and Moony.
